docs/source/specification.md

```diff
@@ -39,23 +39,22 @@ Below we describe each level of the `rawdata` folder hierarchy in more detail. T
 ### Subject
 
 * Each subject *must* have exactly one subject-level folder.
-* Subject-level folders *must* be prefixed with a key-value pair that is unique for each subject. The key *must* be `sub` and the value *must* be numerical, e.g. `sub-001`.
-* Subjects *should* be assigned ascending numerical labels as they are added to the project. The labels *should* be prefixed with an arbitrary number of 0s for consistent indentation and sorting, e.g. `sub-001`, `sub-002`, `sub-003`.
+* Subject-level folders *must* be prefixed with a key-value pair that is unique for each subject. The key *must* be `sub` and the value *must* be alphanumerical, e.g. `sub-123A`.
```

Comment:
I wonder whether we should give a few more examples here, to really drive home what alphanumerical means, e.g. , `sub-123`, `sub-ABC`, `sub-123A`. This makes it explicit that they can use letter, numbers or a combo. Same for sessions.